bedrock/firefox/views.py

```diff
@@ -577,7 +581,12 @@ def get_template_names(self):
                 elif locale == "en-GB":
                     template = "firefox/whatsnew/whatsnew-fx110-uk.html"
                 else:
-                    template = "firefox/whatsnew/whatsnew-fx110-en.html"
+                    if variant == "2":
+                        template = "firefox/whatsnew/whatsnew-fx110-en-features-v1.html"
+                    elif variant == "3":
+                        template = "firefox/whatsnew/whatsnew-fx110-en-features-v2.html"
+                    else:
+                        template = "firefox/whatsnew/whatsnew-fx110-en.html"
```

Comment:
This is a little brain breaking for a monday 🧠

Reply:
My Monday morning brain thinks much clearer than the Friday afternoon brain that wrote this. 🤦